econml/sklearn_extensions/federated_learning.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) PyWhy contributors. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+from econml.dml import LinearDML
+from econml.sklearn_extensions.linear_model import StatsModelsLinearRegression
+from typing import List
+
+
+class FederatedLearner:

```

Comment:
It would probably be better to have this class implement the whole LinearCateEstimator API using the aggregated final model.  However, we could defer that for now if you'd like.

But in any case, there should at least be a docstring that describes the basics of the class, like what its purpose is and what attributes are present.

Reply:
Agreed, we could discuss how we would like to do as implementation of the whole LinearCateEstimator API; For now, I just add the docstring for the class.

Reply:
Related linting should be fixed too.